[TwigComponent] Deprecate `cva` twig function (#2144)
symfony/ux: src/TwigComponent/doc/index.rst

    The ``cva`` function was added in TwigComponents 2.16.

.. deprecated:: 2.20

    The ``cva`` function was deprecated in TwigComponents 2.20, and will be removed in 3.0.

>     The function is now called `html_cva`_ and is now part of ``twig/html-extra:^3.12`` package.

Reviewer: ```suggestion
    The function is now provided by the ``twig/html-extra:^3.12`` package under the name `html_cva`_.
```

?
